src/xml/node.cr

```diff
@@ -477,6 +477,11 @@ struct XML::Node
     LibXML.xmlUnlinkNode(self)
   end
 
+  # Adds the node as the child of this node in the XML document
+  def addChild(child : Node)
```

Comment:
In Crystal (or in the standard library at least) methods have the `snake_case`, not the `camelCase`: (spec needs to be changed too)
```suggestion
  def add_child(child : Node)
```

Reply:
Good catch! Fixed it up. I've been spending too much time in Java land.